handlers/whatsapp/whatsapp.go

```diff
@@ -1046,7 +1046,7 @@ func hasTiersError(payload mtErrorPayload) bool {
 
 func hasWhatsAppContactError(payload mtErrorPayload) bool {
 	for _, err := range payload.Errors {
-		if err.Code == 1006 && err.Title == "Resource not found" && err.Details == "unknown contact" {
+		if err.Code == 1006 && err.Title == "Resource not found" && (err.Details == "unknown contact" || err.Details == "Could not retrieve phone number from contact store") {
```

Comment:
We checking the exact string since the same error code and error title are returned for missing file such as attachment so the details are the only place to make sure we need to check/revalidate the contact from WA API 

WA uses `unknown contact` as details and D3 uses `Could not retrieve phone number from contact store`

Reply:
Seems we shouldn't be checking error message strings no?.. error messages change, error codes should be stable

Reply:
wouldn't the most robust way to check this be `IF channel = D3 and error code = 1006` ?

Reply:
I am pretty sure D3 will also return 1006 for the file not found case as the wrap the WA API

Reply:
Also the details change could be something we the recent WA docker we have not yet tried yet